### PR TITLE
Update features.

### PR DESCRIPTION
**Pull request by EdgarBarrantes**
Updates the features because of changes done in live years ago.

### CODE REVIEW

File: sites/all/modules/features/auge_crumbs/auge_crumbs.strongarm.inc [hunk 1]
```diff
@@ -45,6 +45,7 @@ function auge_crumbs_strongarm() {
   $strongarm->value = array(
     'auge' => 'theme_breadcrumb',
     'seven' => 'theme_breadcrumb',
+    'bartik' => 'theme_breadcrumb',
```
Comment (kporras07) [inline sites/all/modules/features/auge_crumbs/auge_crumbs.strongarm.inc]: You really need bartik enabled?

File: sites/all/modules/features/auge_menus/auge_menus.info [hunk 2]
```diff
@@ -12,7 +12,6 @@ features[menu_links][] = main-menu_blog:/blog
 features[menu_links][] = main-menu_blog:blog
 features[menu_links][] = main-menu_contact:/contact
 features[menu_links][] = main-menu_contacto:/contacto
-features[menu_links][] = main-menu_emprender:/emprender
```
Comment (kporras07) [inline sites/all/modules/features/auge_menus/auge_menus.info:15]: Doing this won't delete the menu item.

Comment (EdgarBarrantes) [reply sites/all/modules/features/auge_menus/auge_menus.info:15]: Yeah it's been deleted in live so it's not really necessary.

**Comment by kporras07** [comment]
:shipit:

**Pull request merged**